tests/e2e/fixture.md

```
@@ -1,50 +1,50 @@
-## 最後の節
+## The last section
 
-長い段落を 1 つ置いておく。折り返しても 1 行目にアンカーが揃うことと、レールの吹き出しが本文の高さを変えないことを確認するために使う。
+One long paragraph, here on purpose. It has to wrap, so that a row taller than one line exists to check two things against: that the anchor stays level with the first line of the row rather than its middle, and that a bubble out on the rail never changes the height of the document itself.
 
-## 生 HTML
+## Raw HTML
 
 <script>window.xssMarker = 'executed';</script>
 
 <img src=x onerror="window.xssMarker = 'executed'">
```

markdownlint-cli2 (0.23.2)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)